src/CommunityToolkit.Maui.Core/Views/Popup/PopupExtensions.windows.cs

```diff
@@ -63,41 +63,60 @@ public static void ConfigureControl(this Popup mauiPopup, IPopup popup, IMauiCon
 	/// </summary>
 	/// <param name="mauiPopup">An instance of <see cref="Popup"/>.</param>
 	/// <param name="popup">An instance of <see cref="IPopup"/>.</param>
-	public static void SetSize(this Popup mauiPopup, IPopup popup)
+	/// <param name="mauiContext">An instance of <see cref="IMauiContext"/>.</param>
+	public static void SetSize(this Popup mauiPopup, IPopup popup, IMauiContext? mauiContext)
 	{
+		ArgumentNullException.ThrowIfNull(mauiContext);
+		ArgumentNullException.ThrowIfNull(popup.Content);
+
 		const double defaultBorderThickness = 0;
 		const double defaultSize = 600;
 
+		var popupParent = mauiContext.GetPlatformWindow();
 		var standardSize = new Size { Width = defaultSize, Height = defaultSize / 2 };
 
-		var currentSize = popup.Size != default ? popup.Size : standardSize;
+		Size currentSize = standardSize;
 
-		if (popup.Content is not null && popup.Size == default)
+		if (!popup.Size.IsZero)
```

Comment:
please invert if

Reply:
@VladislavAntonyuk , I reversed the if condition and swapped the true and false code.
I would like to do the same for the iOS and Android code that was fixed in PR #1361.